www/js/goals.js

```diff
@@ -129,7 +130,35 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
     /*$scope.data = {
         showDelete: false
     };*/
+    $scope.toBrowser = function() {
+            var options = {
+              location: 'yes',
+              clearcache: 'yes',
+              toolbar: 'yes'
+            };
+
+            var settings = localStorage.getItem("habit-mobile-settings")
```

Comment:
where is `habit-mobile-settings` set?
I don't see you setting it anywhere in this PR...